server/errorHandler.ts

```diff
@@ -23,6 +23,9 @@ export default function createErrorHandler(production: boolean) {
 
     res.status(error.status || 500)
 
+    if (res.statusCode === 404) {
+      return res.render('pages/error-404')
+    }
```

Comment:
Errors that are not 404 will still be rendered with the existing error template (which is pretty nasty and ugly); but this should give scope and a way forward for as and when we need to implement the other error content pages